torch/_torch_docs.py

```diff
@@ -181,15 +155,15 @@ def merge_dicts(*dicts):
 .. math::
     \text{{out}} = \text{{input}} + \text{{alpha}} \times \text{{other}}
 
-If :attr:`other` is of type FloatTensor or DoubleTensor, :attr:`alpha` must be
-a real number, otherwise it should be an integer.
+The category (floating point, integer, or Boolean) of :attr:`alpha` must match
+that of the result of addition between :attr:`input` and :attr:`other`.
```

Comment:
This isn't true. the following (integer alpha, float result) works for me on master:
```
import torch

x = torch.randn(3, 3)
y = torch.randn(3, 3)
torch.add(x, y, alpha=1)
```

Reply:
I would change this to say the following:

```
If the result of the addition between :attr:`input` and :attr:`other` is:
- floating point, :attr:`alpha` can be `float` or `int`
- integral, :attr:`alpha` must be `int`
- boolean, :attr:`alpha` can be `int` or `bool.
```

Reply:
Thanks, I reworded a bit and updated this PR.